templates/preempt-rt/preempt-rt.jinja2

```diff
@@ -0,0 +1,30 @@
+- test:
+    timeout:
+      minutes: 10
+
+    definitions:
+    - from: inline
+      repository:
+        metadata:
+          format: Lava-Test Test Definition 1.0
+          name: preempt-rt-prereq
+          description: Pre-requisites for PREEMPT_RT
+        run:
+          steps:
+          - apt-get update && apt-get install -y procps rt-tests
+      name: preempt-rt-prereq
+      path: inline/preempt-rt-prereq.yaml
+
+    - repository: https://github.com/igaw/test-definitions.git
```

Comment:
We should be getting those test definitions from `kernelci/test-definitions` now, with the `kernelci.org` branch.  Let's get a couple of staging jobs to run with this repo first but then we would need to get it changed before deploying in production.

Reply:
I don't think we'll get any runs on staging because there are no rt-stable trees built in staging.  Maybe you add rt-stable v5.4 at least temporarily?

Reply:
Looks like I need to switch to the kernelci repo right now.  Daniel submitted his stuff upstream and removed his `preempt-rt` branch.

Reply:
Starting tree monitor jobs for some stable-rt build configs manually is fine for now, we shouldn't be building that all the time as it's a significant load unless we make a special config for staging or pull from stable-rt into the kernelci staging kernel branch.